styles/_helpers.scss

```diff
@@ -4,5 +4,5 @@
 }
 
 .is-hidden {
-  display: none;
+  display: none !important;
```

Comment:
@nerik I right here.
In the pattern we try to use, they call these kinds of rules "helpers": http://rscss.io/helpers.html